src/roles/wildchild.py

```diff
@@ -1,78 +1,69 @@
-import random
 import re
+import random
+import itertools
+import math
+from collections import defaultdict
 
-import src.settings as var
+import botconfig
```

Comment:
don't import botconfig in role files. It is never needed. If you need botconfig in a role file, you're doing something wrong so change what you're doing so it isn't needed.

Reply:
I just copied the imports from `roles/_skel.py`. I don't use it anywhere.